kubernetes/samples/scripts/common/utility.sh

```diff
@@ -562,3 +562,117 @@ function createDomain {
   # Print a summary
   printSummary
 }
+
+# checks if a given pod in a namespace has been deleted
+function checkPodDelete(){
+
+ pod=$1
+ ns=$2
+ status="Terminating"
+
+ if [ -z ${1} ]; then 
+  echo "No Pod name provided "
+  exit -1 
+ fi
+
+ if [ -z ${2} ]; then 
+  echo "No namespace provided "
+  exit -2 
+ fi
+
+ echo "Checking Status for Pod [$pod] in namesapce [${ns}]"
+ max=10
+ count=1
+ while [ $count -le $max ] ; do
+  sleep 5 
+  pod=`kubectl get po/$1 -n ${ns} | grep -v NAME | awk '{print $1}'`
+  if [ -z ${pod} ]; then 
```

Comment:
@anpanigr, can this code handle the case where the pod is already gone?

Reply:
Yes, if it did not find the pod, it breaks from the loop  

pod=`kubectl get po/$1 -n ${ns} | grep -v NAME | awk '{print $1}'`
  if [ -z ${pod} ]; then
    status="Terminated"
    echo "Pod [$1] removed from nameSpace [${ns}]"
    break;
  fi